Explorer/Assets/DCL/Browser/DecentralandUrls/DecentralandUrlsSource.cs

```diff
@@ -46,8 +46,8 @@ private static string RawUrl(DecentralandUrl decentralandUrl) =>
                 DecentralandUrl.Host => $"https://decentraland.{ENV}",
                 DecentralandUrl.PeerAbout => $"https://peer.decentraland.{ENV}/about",
                 DecentralandUrl.DAO => $"https://decentraland.{ENV}/dao/",
-                DecentralandUrl.Notification => $"https://notifications.decentraland.{ENV}/notifications",
-                DecentralandUrl.NotificationRead => $"https://notifications.decentraland.{ENV}/notifications/read",
+                DecentralandUrl.Notification => $"https://notifications.decentraland.zone/notifications",
+                DecentralandUrl.NotificationRead => $"https://notifications.decentraland.zone/notifications/read",
```

Comment:
I guess you're currently hardcoding `.zone` here for testing purposes, right? We should change it before going to `main`.
```suggestion
                DecentralandUrl.Notification => $"https://notifications.decentraland.{ENV}/notifications",
                DecentralandUrl.NotificationRead => $"https://notifications.decentraland.{ENV}/notifications/read",
```

Reply:
yes it's for QA to test and will be reverted as soon as they approve it, as it happened in [this](https://github.com/decentraland/unity-explorer/pull/2148) PR